deployment/kubernetes/google-kubernetes-engine/bookie.yaml

```diff
@@ -155,6 +156,29 @@ spec:
             storage: 10Gi
         storageClassName: bookie-hdd
 
+
+---
+##
+## Define the Bookie headless service
+## In practice, in this case, it is only useful to have a view of
+## all the bookie pods that are present
```

Comment:
Can you also mention this is needed to have K8S to assign routable DNS names to the bookies pods (eg: to have `bookkeeper-2.bookkeeper.default.svc.cluster.local` as a routable "hostname")

Reply:
Sure. I copied the comment from the generic k8s service definition, but I pushed a commit to make it more obvious.